### DISCUSSION TITLE
Close debugger windows when closing the Debugger

### DISCUSSION THREAD

**Discussion by lukiz84** [TabularEditor/TabularEditor3#1034] [Ideas]
Hi,

it would be nice to automatically close all the debugger windows (watch, local, calltree etc.) when closing the debugger.

thx and br

**Comment by DBojsen** [comment]
Hi @lukiz84 

We have added this to our backlog - for now you can use the ability to save/load window layouts: https://docs.tabulareditor.com/te3/features/user-interface.html#customizing-the-user-interface

BR David - TE3 Support

**Comment by corycundy** [comment]
Hello,

I wanted to add another point of view. I often add many watches to the debugger window over the course of a session and wouldn't want them to go away if I accidentally closed the Debugger windows. When reviewing this item, would you consider making this an option vs. something that happens all the time?

Thanks for the consideration.

Cory

**Comment by DBojsen** [comment]
Hi @corycundy 

It makes a lot of sense.  Added to backlog as well.
Next time, please create a separate issue and reference the original one, as we use the individual issues to track each feature request. ;-)

BR David - TE3 Support
